trappy/plotter/ILinePlotGen.py

```diff
@@ -210,7 +210,7 @@ def add_plot(self, plot_num, data_dict, title="", test=False):
 
         fig_name = self._fig_map[plot_num]
         fig_params = {}
-        fig_params["data"] = OrderedDict((k, v.T.to_dict()) for k, v in data_dict.iteritems())
+        fig_params["data"] = data_dict
```

Comment:
`v.T.to_dict()` is necessary for the for values that are specified as a series instead of a dict.  Try to plot one column of a dataframe.  I'm not sure how to fix this cleanly though.